sycl/include/sycl/accessor.hpp

```diff
@@ -1144,9 +1141,18 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(accessor) accessor :
   }
 
   // __init variant used by the device compiler for ESIMD kernels.
-  // TODO In ESIMD accessors usage is limited for now - access range, mem
+  // TODO: In ESIMD accessors usage is limited for now - access range, mem
```

Comment:
memory/access range and offset is not supported because it is not even passed to __init_esimd().
That should be a separate effort to enable those fields for accessors.

I even tried to _fully_ support local accessors, but met some unexpected difficulties that couldn't resolve in 1.5 days, then did a step back and left it for future work.

Reply:
if they arent supported why do we need to init them to 0?

Reply:
They may be used in get_pointer() or operator[] API. They should not have garbage.

Reply:
ok so the comment is talking about api use by customer, not internal, got it

Reply:
Hmm, actually, my argument is correct for offset, regarding the memory/access range being initialized with zeroes, I need to re-check something. Let me return to this question real quick.

Reply:
With this patch only 1-d buffer accessors are supported. 
2-d buffers accessors remain unsupported - they would require the access-range to be passed and initialized correctly (not with zeroes).